app/assets/stylesheets/finder_frontend.scss

```diff
@@ -66,6 +61,18 @@
   }
 }
 
+.document--top {
+  border: 1px solid $grey-3;
```

Comment:
Could you replace `$grey-3` with `govuk-colour("grey-3")` we are trying to move more towards govuk-frontend and move away for govuk-elements https://design-system.service.gov.uk/styles/colour/